### PR TITLE
docs: add migration guide from v6 to v7

### PR DESCRIPTION
**Pull request by matthewmayer**
I saw there was a migration guide from v5 to v6

https://fakerjs.dev/guide/migration-guide-v5.html

But no such guide for the breaking changes from v6 to v7 described at https://github.com/faker-js/faker/releases/tag/v7.0.0

I thought it would be useful to add this as a reference, as I recently upgraded a project and had to do some digging to find details of the changes.

### CODE REVIEW

File: docs/guide/migration-guide-v6.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Migrating from Faker v6 to v7
+
+[[toc]]
+
+### Node 12 no longer supported
+
+You require at least Node 14 to run faker
```
Comment (ST-DDT) [inline docs/guide/migration-guide-v6.md:7]:
```suggestion
You need at least Node 14 to use faker.
```

File: docs/guide/migration-guide-v6.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+### Default export removed
+
+If you were previously importing faker like this:
+
+```js
+import faker from '@faker-js/faker';
+```
+
+You should instead use
```
Comment (ST-DDT) [inline docs/guide/migration-guide-v6.md:17]:
```suggestion
You should instead use:
```

File: docs/guide/migration-guide-v6.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+```js
+import { faker } from '@faker-js/faker';
+```
+
+### Deprecated methods changed
+
+Several methods were renamed or moved.
```
Comment (ST-DDT) [inline docs/guide/migration-guide-v6.md:25]: Not sure about this one:

```suggestion
Several methods were renamed or moved:
```

**Comment by ST-DDT** [comment]
Thank you a lot, this migration guide is really overdue.

**Comment by ST-DDT** [comment]
<details>
<summary><strong>Docs Preview</strong> (Click to expand)</summary>

grafik


</details>

**Review by Minozzzi — DISMISSED** [review]

**Review by ST-DDT — DISMISSED** [review]

File: docs/.vitepress/config.ts [hunk 4]
```diff
@@ -139,6 +139,10 @@ export default defineConfig({
               text: 'Migrating from Faker v5',
               link: '/guide/migration-guide-v5',
             },
+            {
+              text: 'Migrating from Faker v6',
+              link: '/guide/migration-guide-v6',
```
Comment (Shinigami92) [inline docs/.vitepress/config.ts:144]: For now we can make it like this, but at Vite, we use only one migration file and use the version switch at the top to redirect the user to the older version of the documentation that then include the migration guide of the older version to the old version

File: docs/guide/migration-guide-v6.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+# Migrating from Faker v6 to v7
+
+[[toc]]
+
+### Node 12 no longer supported
+
+You need at least Node 14 to use faker.
```
Comment (Shinigami92) [inline docs/guide/migration-guide-v6.md:7]:
```suggestion
You need at least Node 14 to use Faker.
```

File: docs/guide/migration-guide-v6.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+### Default export removed
+
+If you were previously importing faker like this:
+
+```js
+import faker from '@faker-js/faker';
+```
+
+You should instead use:
```
Comment (Shinigami92) [inline docs/guide/migration-guide-v6.md:17]:
```suggestion
You must instead use:
```

**Review by Shinigami92 — CHANGES_REQUESTED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Review by ejcheng — APPROVED** [review]

**Pull request merged**